chart-modules/common/picasso/chart-builder/presets/dimension-measure-chart/dimension-measure-chart.js

```diff
@@ -155,6 +156,7 @@ function addMeasureAxisComponent(chartBuilder, options, measureDirection, measur
 function addMeasureAxisTitleComponent(chartBuilder, options, measureDirection) {
   let measureTitleSettings = {
     text: options.measureTitleText,
+    style: options.style || {},
```

Comment:
I would like a more specific name. (not just style)
(maybe something like axisTitleStyle)
(or maybe just have both this and new axisLabelSyle/setting with under style)

Reply:
Changed the names to match the styles.